domain-models-runtime/src/main/java/org/folio/rest/persist/PgUtil.java

```diff
@@ -1094,7 +1095,7 @@ private static String getAscDesc(ModifierSet modifierSet) {
         ascDesc = "DESC";
         break;
       default:
-        // ignore
+        throw new CQLFeatureUnsupportedException("CQL: Unsupported modifier " + modifier.getType());
       }
```

Comment:
Wonder why this is not covered in the tests. It looks as if it should be.

Reply:
The wrong sortBy column caused the non-optimized code path been executed.